finality-aleph/Cargo.toml

```diff
@@ -6,41 +6,41 @@ edition = "2021"
 
 [dependencies]
 aleph-primitives = { package = "primitives", path = "../primitives" }
-tokio = { version = "1.15.0", features = [ "sync", "macros", "time", "rt-multi-thread" ] }
-codec = { package = "parity-scale-codec", version = "2", default-features = false, features = ["derive"] }
+tokio = { version = "1.17", features = [ "sync", "macros", "time", "rt-multi-thread" ] }
+codec = { package = "parity-scale-codec", version = "3.0", default-features = false, features = ["derive"] }
 hash-db = { version = "0.15.2", default-features = false }
-log = "0.4"
-lru = "0.6"
-rand = "0.8"
-futures = "0.3"
+log = "0.4.16"
```

Comment:
do we need here to be so specific? also for `futures`

Reply:
I reverted these changes, I think they were made by some tool